packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -315,16 +318,23 @@ export class WalletService {
           return cb(new ClientError(Errors.codes.NOT_AUTHORIZED, 'Copayer not found'));
         }
 
-        if (!copayer.isSupportStaff) {
+        if (!copayer.isSupportStaff && !copayer.isMarketingStaff) {
```

Comment:
Probably no need to change anything here. WalletId should just be the one lf the orginal copayer if 'isMarketingStaff`